src/vs/platform/actions/browser/menusExtensionPoint.ts

```diff
@@ -30,6 +30,7 @@ namespace schema {
 			case 'editor/title': return MenuId.EditorTitle;
 			case 'editor/context': return MenuId.EditorContext;
 			case 'explorer/context': return MenuId.ExplorerContext;
+			case 'editor/tab': return MenuId.EditorTab;
```

Comment:
also this needs to be exposed in the [schema](https://github.com/Microsoft/vscode/blob/master/src/vs/platform/actions/browser/menusExtensionPoint.ts#L86) etc